README.md

```diff
@@ -1,34 +1,71 @@
-# Membrane Multimedia Framework: Template Plugin
+# Membrane Realtimer Plugin
```

Comment:
```suggestion
# Membrane Realtimer plugin
```
In all repos we have `Membrane Xyz plugin`. `Membrane Xyz Plugin` looks better imo but let's follow the convention.